src/factories/routePathFactory.ts

```diff
@@ -17,43 +17,50 @@ class RoutePathFactory {
         const internalRoutePathId = HashHelper.getHashFromString(
             [
                 routeId,
-                suunta.suunimi,
                 suunta.suuvoimast,
-                suunta.suuvoimviimpvm,
                 suunta.suusuunta,
             ].join('-'),
         ).toString();
 
-        const routePathLinkResult:IRoutePathLinkResult[]
-        = suunta.reitinlinkkisByReitunnusAndSuuvoimastAndSuusuunta.edges
+        const routePathLinkResult:IRoutePathLinkResult[] | null
+        = suunta.reitinlinkkisByReitunnusAndSuuvoimastAndSuusuunta ?
+            suunta.reitinlinkkisByReitunnusAndSuuvoimastAndSuusuunta.edges
```

Comment:
Making a little tweak to the query would allow removal of .edges and .node. Not sure which way we want to do it though. Personally I find this edges-node less intuitive than using .nodes

Reply:
Yes, we probably use this at many places, we could refactor them to using simpler form, but maybe not in this task